apps/velo-external-db/src/app.ts

```diff
@@ -1,16 +1,18 @@
-const express = require('express')
-const { create, readCommonConfig } = require('@wix-velo/external-db-config')
-const { ExternalDbRouter } = require('@wix-velo/velo-external-db-core')
-const { engineConnectorFor } = require('./storage/factory')
+import express = require('express')
+import { create, readCommonConfig } from '@wix-velo/external-db-config'
+import { ExternalDbRouter } from '@wix-velo/velo-external-db-core'
+import { engineConnectorFor } from './storage/factory'
+import { Server } from 'http'
+import { ConnectionCleanUp, ISchemaProvider } from '@wix-velo/velo-external-db-types'
+import { Hooks } from 'libs/velo-external-db-core/src/types'
```

Comment:
@Idokah importing from another package via path and not via the package is really bad idea .....